model/registry/gen-ai.yaml

```diff
@@ -71,6 +71,12 @@ groups:
         brief: The number of tokens used in the LLM response (completion).
         examples: [180]
         tag: llm-generic-response
+      - id: usage.token_type
+        stability: experimental
+        type: string
```

Comment:
should it be an enum?

Reply:
I made it an enum with ability to provide custom value.